ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -13,6 +13,14 @@ base_domain=$(<"${SHARED_DIR}"/basedomain.txt)
 declare -a vips
 mapfile -t vips < "${SHARED_DIR}/vips.txt"
 
+cat > "${SHARED_DIR}"/.awsconf <<EOF
+[profile default]
+region = us-east-1
+output = json
+retry_mode = adaptive
+max_attempts = 7
```

Comment:
does this retry RateLimitExceeded errors? Because it is possible that cli is not treating them as retryable error.

Reply:
@cuppett do you know? ^